index.html

```diff
@@ -0,0 +1,761 @@
+                                    <span>Cooking</span>
+                                </div>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Email Newsletters</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Corporate Subscriptions</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Education Rate</span>
+                            </li>
+                        </ul>
+                        <ul class="itemsSubNavInfo topicList">
+                            <li class="normalItemSubNav topicList">
+                                <span>Mobile Application</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Replica Edition</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>International</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Canada</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Español</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>中文网</span>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <footer id="mainFooter">
+        <ul id="listMainFooter">
+            <li class="footerItem">© 2021 The New York Times Company</li>
+            <li class="footerItem">NYTCo</li>
+            <li class="footerItem">Contact Us</li>
+            <li class="footerItem">Accessibility</li>
+            <li class="footerItem">Work with us</li>
+            <li class="footerItem">Advertise</li>
+            <li class="footerItem">T Brand Studio</li>
+            <li class="footerItem">Your Ad Choices</li>
+            <li class="footerItem">Privacy Policy</li>
+            <li class="footerItem">Terms of Service</li>
+            <li class="footerItem">Site Map</li>
+            <li class="footerItem">Help</li>
+            <li class="footerItem">Subscriptions</li>
```

Comment:
These items should be linked, using `#` as the value for their `href` attributes.